src/bluehawk/actions/snip.ts

```diff
@@ -122,6 +132,19 @@ export const formatInRst = async (
   return formattedSnippet;
 };
 
+export const formatInMd = (result: ProcessResult): string | undefined => {
+  const { document } = result;
+  const snippet = document.attributes["snippet"];
+  if (snippet === undefined) {
+    return undefined;
+  }
+  const language = document.extension.slice(1);
```

Comment:
Hrm, this probably works for a lot of cases, but `result.parseResult.languageSpecification` might contain what you want more accurately

Reply:
this actually doesn't work. `result.parseResult.languageSpecification` is an object, and `result.parseResult.languageSpecification?.languageId` returns `C-like`.